client/__tests__/e2e/cellxgeneActions.ts

```diff
@@ -43,262 +37,256 @@ export async function drag(testId: any, start: any, end: any, lasso = false) {
   await page.mouse.up();
 }
 
-export async function scroll({testId, deltaY, coords}: {testId: string, deltaY: number, coords: number[]}) {
-  const layout = await waitByID(testId);
-  if (layout){
+export async function scroll({
+  testId,
+  deltaY,
+  coords,
+  page,
+}: {
+  testId: string;
+  deltaY: number;
+  coords: number[];
+  page: Page;
+}): Promise<void> {
+  const layout = await page.getByTestId(testId);
```

Comment:
no need to await here, since Playwright locator is designed to check the DOM at the time of user action.

E.g.,

```ts
const button = page.getByTestId("test-button");
await button.click() // this we will await!
```

Reply:
check to see what layout is and why we need to wait and what we should be waiting for (visible?)